apps/dashboard/app/apps/ood_app.rb

```diff
@@ -264,7 +264,8 @@ def run_setup_production
           #
           # This makes the execution of the setup-production script use the same ruby versions
           # that Passenger uses when launching the app.
-          output, status = Open3.capture2e({'PATH' => path.join('bin').to_s + ':'+ ENV['PATH']}, 'bundle','exec', setup)
+          cmd = File.exist?('./Gemfile') ? "bundle exec #{setup}" : setup
+          output, status = Open3.capture2e({'PATH' => path.join('bin').to_s + ':'+ ENV['PATH']}, cmd)
```

Comment:
Sorry to be a little nit-picky about this - but we really should be passing Arrays to `Open3.capture2e` instead of 1 single string. The reason is security related in that arrays of strings will be escaped when they issue the command whereas 1 single string won't.

This won't happen here because the variable `setup` is hard coded - but even so we should just be in the practice of using arrays of strings instead of 1.

For context - if `setup` were a variable, one could set that to `/bin/some-nefarious-command; ./bin/setup-production` and this would actually execute the nefarious-command _in addition to_ `./bin/setup-production`.